### PR TITLE
add fatality rate graph

### PR DESCRIPTION
**Pull request by andy-held**
list with populations by country crawled from wikipedia

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,6 +124,12 @@
     <script>
         "use strict"
 
+        var population = {}
+        fetch('./population_from_wikipedia.json')
```
Comment (paroj) [inline index.html]: this could finish after our other fetch

File: population_from_wikipedia.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
+{"China": 1401855920,
```
Comment (paroj) [inline population_from_wikipedia.json]: if you write `var population = ...` here you can, just `<script src="` it.

**Review by paroj — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -152,6 +163,14 @@
             return mortality
         }
 
+        function getDeadPerMillion(data, country) {
+            var dead = []
+            for (var day of data[country]) {
+                if (day["deaths"] < 1) continue
```
Comment (paroj) [inline index.html]: I would prefer "Days since 25 confirmed" to keep it comparable with other graphs

**Comment by andy-held** [comment]
can we test this before merging?

**Comment by paroj** [comment]
push it to your master?

File: population_from_wikipedia.js [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+"Afghanistan": 32225560,
+"Venezuela": 32219521,
+"Peru": 32131400,
+"Angola": 31127674,
+"Ghana": 30280811,
+"Mozambique": 30066648,
+"Yemen": 29825968,
+"Nepal": 29609623,
+"Cameroon": 26545864,
+"Ivory Coast": 25823071,
```
Comment (paroj) [inline population_from_wikipedia.js]: this should be `Cote d'Ivoire`

File: population_from_wikipedia.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+"Libya": 6871287,
+"Lebanon": 6825442,
+"Kyrgyzstan": 6523500,
+"El Salvador": 6486201,
+"Nicaragua": 6460411,
+"Turkmenistan": 6031187,
+"Denmark": 5822763,
+"Singapore": 5703600,
+"Finland": 5527573,
+"Congo": 5518092,
```
Comment (paroj) [inline population_from_wikipedia.js]: `Congo (Kinshasa)` or `Congo (Brazzaville)`? There is no Congo in JHU data

Comment (andy-held) [reply population_from_wikipedia.js]: Yeah, they were just crawled from Wiki, I went through some obvious differences, but some are hard...

Comment (andy-held) [reply population_from_wikipedia.js]: Adjusted everything to match JHU data.

**Comment by andy-held** [comment]
Also tested it.

**Comment by paroj** [comment]
:+1:

**Pull request merged**